test/services/videoconference/index.test.js

```diff
@@ -1,15 +1,13 @@
 /* eslint-disable global-require */
-const { Configuration } = require('@schul-cloud/commons');
+/* const { Configuration } = require('@schul-cloud/commons');
 const { warning } = require('../../../src/logger');
 
-const Config = new Configuration();
-Config.init(); // todo use singleton instead
-
-const FEATURE_VIDEOCONFERENCE_ENABLED = Config.get('FEATURE_VIDEOCONFERENCE_ENABLED');
+const FEATURE_VIDEOCONFERENCE_ENABLED = Configuration.get('FEATURE_VIDEOCONFERENCE_ENABLED');
 
 if (FEATURE_VIDEOCONFERENCE_ENABLED === true) {
 	require('./index');
 	require('./logic/index');
 } else {
 	warning('do not execute bbb tests, due FEATURE_VIDEOCONFERENCE_ENABLED has not been enabled');
 }
+ */
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Newline required at end of file but not found.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4991925)